lib/iron_bank/describe/excluded_fields.rb

```diff
@@ -79,8 +81,8 @@ def remove_last_failure_fields
 
       def valid_query?
         # Querying using the ID (indexed field) should return an empty
-        # collectio fast when it's successful
-        object.where(id: "XYZ")
+        # collection fast when it's successful
+        object.where(id: INVALID_OBJECT_ID)
```

Comment:
Shouldn't the `frozen_string_literal` take care of not having to create a specific constant instead?

```suggestion
        object.where(id: "InvalidObjectId")
```

Reply:
Never mind, saw that you are using the constant in the spec.

👍